index.bs

```diff
@@ -49,29 +49,45 @@ conversion domain.
 
 # HTML monkeypatches # {#html-monkeypatches}
 
-Add the following content attributes to the <{a}> element:
+Add the following <a spec=html>content attributes</a> to the <{a}> element:
```

Comment:
We should file a bug against HTML to check if there's a better way to do this than referring to a bunch of non-exported `dfn`s. E.g. HTML could export these definitions.

I believe Anne also likes us to file an HTML issue saying that this monkeypatch exists, like https://github.com/whatwg/fetch/issues/784.

Reply:
In general monkeypatch specs need a lot of similar references to non-exported definitions; that's probably the best way.